modin/pandas/test/test_io.py

```diff
@@ -60,8 +62,21 @@
 TEST_GBQ_FILENAME = "test_gbq."
 SMALL_ROW_SIZE = 2000
 
+test_data_dir = os.path.join(os.path.dirname(__file__), "data")
```

Comment:
The name should let us know that this folder can be deleted.

Reply:
This is common directory for test data that is already used by other tests. Do you mean we need to create separate directory for `read_csv` tests?

Reply:
Yes, do not mix the generated data with source data from github.

Reply:
Ok, separate directory for `read_csv` tests data is used now.